lua/starfall/sflib.lua

```diff
@@ -951,6 +951,20 @@ function SF.CheckRagdoll(model)
 	return false
 end
 
+local drawEntityClasses = {
+	["prop_physics"] = true,
+	["prop_ragdoll"] = true,
+	["prop_vehicle_jeep"] = true,
+	["prop_vehicle_airboat"] = true,
+	["prop_vehicle_prisoner_pod"] = true,
+}
+function SF.CanDrawEntity(ent)
+	if not drawEntityClasses[ent:GetClass()] then return false end
```

Comment:
`return drawEntityClasses[ent:GetClass()] and ent.RenderOverride==nil and not ent:IsEffectActive(EF_BONEMERGE)`